src/Modules/OrchardCore.Commerce/Services/ShoppingCartHelpers.cs

```diff
@@ -83,7 +83,10 @@ public async Task<ShoppingCartItem> ParseCartLineAsync(ShoppingCartLineUpdateMod
         {
             await _notifier.AddAsync(
                 NotifyType.Error,
-                T["Can't add product {0} because it doesn't have a price.", line.ProductSku]);
+                T[
+                    "Can't add product {0} because it doesn't have a price, or its currency doesn't match with the " +
+                    "current display currency.",
+                    line.ProductSku]);
```

Comment:
The localization string must be on the same line, otherwise [the tooling](https://docs.orchardcore.net/en/dev/docs/reference/modules/Localize/#extract-translations-to-po-files) won't recognize it.

![image](https://user-images.githubusercontent.com/4007293/182474477-b0e6b427-95b2-43b8-8d06-5df09502e59a.png)